data/mods/My_Sweet_Cataclysm/sweet_harvest.json

```diff
@@ -47,7 +47,14 @@
     "message": "As you butcher the chocolate corpse, you break the large chunks of flesh down into bars.",
     "entries": [
       { "drop": "chocolate", "type": "flesh", "mass_ratio": 0.52 },
-      { "drop": "cow_candy", "base_num": [ 0, 2 ], "max": 2, "type": "offal" }
+      { "drop": "cow_candy", "base_num": [ 0, 2 ], "max": 2, "type": "offal" },
+      {
```

Comment:
You've added this to the drop from chocolate cow, that's probably the source of your crash

Reply:
oops, gonna fix that.

Reply:
Don't know why it's going to chocolate cow drops.